### PR TITLE
Translate core strings

### LINKED ISSUE #209: "N member(s)" not translated
screenshot from 2018-10-24 02-14-31

### LINKED ISSUE #224: "Archived chats (n)" not translated
screenshot from 2018-10-26 11-48-00

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes #224, closes #209

### CODE REVIEW

File: _locales/en.json [hunk 1]
```diff
@@ -522,5 +522,157 @@
...
+    "description": ""
+  },
+  "DC_STR_CANNOT_LOGIN": {
+    "message": "Cannot login as %1$s.",
+    "description": ""
+  },
+  "DC_STR_SERVER_RESPONSE": {
+    "message": "Response from %1$s: %2$s",
+    "description": ""
+  }
```
Comment (ralphtheninja) [inline _locales/en.json]: I'm leaving the descriptions empty for now until they have been documented in core.

File: src/main/ipc.js [hunk 2]
```diff
@@ -12,41 +12,40 @@ const menu = require('./menu')
...
+    const p = require.resolve('../../conversations/build/manifest.css')
     e.returnValue = fs.readFileSync(p).toString()
   })
 
-  // Our wrapper for controlling deltachat instances
-  const dc = new DeltaChat(cwd)
-
   // Create a new instance
   ipc.on('login', (e, ...args) => {
-    dc.login(...args, render, onError)
```
Comment (ralphtheninja) [inline src/main/ipc.js]: This `onError` was never called from `dc.login()`

File: src/main/ipc.js [hunk 3]
```diff
@@ -96,3 +91,49 @@ function init (cwd) {
...
+  strings[C.DC_STR_MSGGRPIMGDELETED] = tx('DC_STR_MSGGRPIMGDELETED')
+  strings[C.DC_STR_E2E_PREFERRED] = tx('DC_STR_E2E_PREFERRED')
+  strings[C.DC_STR_ARCHIVEDCHATS] = tx('DC_STR_ARCHIVEDCHATS')
+  strings[C.DC_STR_STARREDMSGS] = tx('DC_STR_STARREDMSGS')
+  strings[C.DC_STR_AC_SETUP_MSG_SUBJECT] = tx('DC_STR_AC_SETUP_MSG_SUBJECT')
+  strings[C.DC_STR_AC_SETUP_MSG_BODY] = tx('DC_STR_AC_SETUP_MSG_BODY')
+  strings[C.DC_STR_SELFTALK_SUBTITLE] = tx('DC_STR_SELFTALK_SUBTITLE')
+  strings[C.DC_STR_CANTDECRYPT_MSG_BODY] = tx('DC_STR_CANTDECRYPT_MSG_BODY')
+  strings[C.DC_STR_CANNOT_LOGIN] = tx('DC_STR_CANNOT_LOGIN')
+  strings[C.DC_STR_SERVER_RESPONSE] = tx('DC_STR_SERVER_RESPONSE')
```
Comment (ralphtheninja) [inline src/main/ipc.js]: This can most likely be refactored later.

**Comment by ralphtheninja** [comment]
Conflicts of course. Rebasing.

**Comment by r10s** [comment]
lgtm :)

wrt strings as such, i hope we'll get an answer if we want to share the same strings/translations for all apps these days :)

**Pull request merged**